src/home/home.scss

```diff
@@ -47,29 +47,34 @@
       }
 
       .buttons {
-        display: flex;
+        display: grid;
+        grid-auto-flow: column;
         justify-content: space-between;
         cursor: pointer;
         width: 710px;
-
+        grid-gap: 24px;
```

Comment:
this affects both column and row.  If the intent here is to only set the column (which I think it is), then please use column-gap.

Reply:
I use grid-gap here intentionally. It serves the gap for both desktop and mobile view (when it is wrapped).